data.sql

```diff
@@ -15,4 +17,31 @@ VALUES
   (8, 'Angemon', '2005-06-12', 1, true, -45),
   (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
   (10, 'Blossom', '1998-11-13', 3, true, 17),
-  (11, 'Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+  (11, 'Ditto', '2022-05-14', 4, true, 22);
+
+-- Insert the following data into the owners table:
+INSERT INTO owners(full_name, age) 
+VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+-- Insert the following data into the species table:
+INSERT INTO species(name) 
+VALUES
+  ('Pokemon'),
+  ('Digimon');
+
+-- Modify your inserted animals so it includes the species_id value:
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE name NOT LIKE '%mon';
+
+-- Modify your inserted animals to include owner information (owner_id):
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
```

Comment:
* [OPTIONAL] This insertion could have been done dynamically where you do not know the `species_id` and  `owner_id`. example:

```sql
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';

   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   ...
```